django/cantusdb_project/main_app/templates/source_create.html

```diff
@@ -57,7 +57,7 @@ <h3>Create Source</h3>
                 <div class="form-row">
                     <div class="form-group m-1 col-lg-3">
                         <label for="{{ form.shelfmark.id_for_label }}">
-                            <small>Shelfmark:</small>
+                            <small>Shelfmark:</small><span class="text-danger" title="This field is required">*</span></small>
```

Comment:
See: https://github.com/DDMAL/CantusDB/issues/1550

Better would be to add the `.small` class to the `<label>` element.

Reply:
I think I've fixes the ones for these fields by using `class='form-control-sm` on the `<label>` element. But there's many more of these cases we'll want to fix in the future.